manifest.json

```diff
@@ -9,5 +10,14 @@
    "fullscreen": {
       "allow": true,
       "open": false
+   },
+   "entities": [
```

Comment:
This just allows the Cloud App framework to recommend the app if users or loans are displayed